pkg/artifactory/resource/repository/remote/resource_artifactory_remote_java_repository.go

```diff
@@ -13,13 +13,13 @@ func ResourceArtifactoryRemoteJavaRepository(repoType string, suppressPom bool)
 			Type:        schema.TypeBool,
 			Optional:    true,
 			Default:     false,
-			Description: `(Optional) When set, if a POM is requested, Artifactory attempts to fetch the corresponding jar in the background. This will accelerate first access time to the jar when it is subsequently requested. Default value is 'false'.`,
+			Description: `When set, if a POM is requested, Artifactory attempts to fetch the corresponding jar in the background. This will accelerate first access time to the jar when it is subsequently requested. Default value is 'false'.`,
```

Comment:
Why is `(Optional)` removed in this case? In other files, `(Optional)` is added to `Description`

Reply:
Then I need to remove (Optional) from all .go files